modules/default/weather/providers/envcanada.js

```diff
@@ -576,32 +587,6 @@ WeatherProvider.register("envcanada", {
 			return kilo;
 		}
 	},
-	//
-	// Convert cm or mm to inches
-	//
-	convertPrecipAmt(amt, units) {
```

Comment:
Are those used nowhere?

Reply:
That's correct. Conversion of precipitation amount and unit-of-measure was moved into the setPrecipitation function as part of cleaning up the bug I found. I think this is cleaner code since setPrecipitation already has the overall responsibility to determine how precipitation will be expressed based on the way the Env Canada service decides to provide the data - (can be an accumulation amount (mm or cm) and/or POP depending on their forecast).